### DISCUSSION TITLE
Langage Selection 3.3.0

### DISCUSSION THREAD

**Discussion by Yopler** [HandyOrg/HandyControl#1130] [Q&A] [answered]
Hi 🖐,

Since the 3.3.0, I strated seeing in my WPF UI some chinese whereas I have set the language to English `ConfigHelper.Instance.SetLang("en");` .
Here an example, in my UI : 

![image](https://user-images.githubusercontent.com/45144437/156006329-8413b1f4-9515-4e90-bfb4-e75c9b0f3cbb.png)

Is the way to select the langage has changed ?
Is it base of the `System.Globalization.CultureInfo.CurrentCulture` , that I have set on InvariantCulture?

**Comment by ghost1372** [comment, marked as answer]
Hi @Yopler 
No, but we have separated the language packs from HandyControl.
The method of changing the language is the same as in the past, But you have to install your desired language pack from Nuget

English pack for HandyControl:
https://www.nuget.org/packages/HandyControl.Lang.en/

English pack for HandyControls:
_HandyControls default language is English you dont need to install (english) language pack_

**Reply by Yopler** [reply]
Thanks !
Does the HandyControl.Lang.en contains only the English langage or also all HandyControl control ans style ?

**Reply by ghost1372** [reply]
> Thanks !
> Does the HandyControl.Lang.en contains only the English langage or also all HandyControl control ans style ? 

HandyControl.Lang.en only contain english  language for other languages you need to install other packages like
HandyControl.Lang.fa
All controls and styles available in core package 
Nuget HandyControl

**Reply by Yopler** [reply]
Ok ! Thanks ! That 's answering my question 😊

**Comment by o-medizadeh-Ard** [comment]
the Growl with prameter 'ShowPersianDateTime = true,' has the following error on date '05/18/2024' :
System.ArgumentOutOfRangeException: 'Year, Month, and Day parameters describe an un-representable DateTime.'
But it works correctly on other dates.
